Shared/Samples/Show viewshed from geoelement in scene/ShowViewshedFromGeoelementInSceneView.swift

```diff
@@ -0,0 +1,194 @@
+            
+            // Set up the heading expression for the tank.
+            let renderer3D = SimpleRenderer()
+            let sceneProperties = RendererSceneProperties(headingExpression: "[heading] + 90", pitchExpression: "[pitch]", rollExpression: "[roll]")
+            sceneProperties.headingExpression = "[HEADING]"
+            renderer3D.sceneProperties = sceneProperties
+            graphicsOverlay.renderer = renderer3D
+            
+            return graphicsOverlay
+        }()
+        
+        /// The analysis overlay for the tank's viewshed.
+        let analysisOverlay = AnalysisOverlay()
+        
+        /// The graphic for the tank.
+        let tankGraphic: Graphic = {
+            let tankSymbol = ModelSceneSymbol(url: .bradleyTank, scale: 10.0)
+            tankSymbol.heading = 90.0
+            tankSymbol.anchorPosition = .bottom
+            let tankGraphic = Graphic(
+                geometry: Point(x: -4.506390, y: 48.385624, spatialReference: .wgs84),
+                attributes: ["HEADING": 0.0],
+                symbol: tankSymbol
+            )
+            return tankGraphic
+        }()
+        
+        /// The point for the tank to move toward.
+        var waypoint: Point!
+        
+        /// The timer for the moving tank animation.
+        var animationTimer: Timer!
+        
+        init() {
+            // Create camera controller.
+            cameraController = OrbitGeoElementCameraController(
+                target: tankGraphic,
+                distance: 200.0
+            )
+            
+            // Add tank graphic to graphics overlay.
+            graphicsOverlay.addGraphic(tankGraphic)
+            
+            // Create a viewshed to attach to the tank.
+            let geoElementViewshed = GeoElementViewshed(
+                geoElement: tankGraphic,
+                horizontalAngle: 90.0,
+                verticalAngle: 40.0,
+                headingOffset: 0.0,
+                pitchOffset: 0.0,
+                minDistance: 0.1,
+                maxDistance: 250.0
+            )
+            
+            // Offset viewshed observer location to top of tank.
+            geoElementViewshed.offsetZ = 3.0
+            
+            // Add the viewshed to the analysisOverlay to add to the scene.
+            analysisOverlay.addAnalysis(geoElementViewshed)
+        }
+        
+        /// Animate the tank moving from its current point to the waypoint.
+        func animate() {
+            // Get point from the current tank position.
+            guard let tankLocation = tankGraphic.geometry as? Point else { return }
+            
+            // Get the distance from the current tank location to the waypoint.
+            guard let distanceResult = GeometryEngine.geodeticDistance(
+                from: tankLocation,
+                to: waypoint,
+                distanceUnit: LinearUnit.meters,
+                azimuthUnit: AngularUnit.degrees,
```

Comment:
Our style is to use implicit member expressions whenever possible.

```suggestion
                distanceUnit: .meters,
                azimuthUnit: .degrees,
```